coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 90.18
```

Comment:
Why the coverage drop (trying to investigate)

Reply:
Seems to be something happening with the helpers

Reply:
it happened with the rails helper :(

Reply:
More specifically the view helpers aren't being tested yet all the tests pass, which shouldn't work since these are view tests. The root cause seems to be from a change in the spec_helper and/or rails_helper.